python/cudf/cudf/core/resample.py

```diff
@@ -1,4 +1,4 @@
-# SPDX-FileCopyrightText: Copyright (c) 2021-2024, NVIDIA CORPORATION & AFFILIATES.
+# SPDX-FileCopyrightText: Copyright (c) 2021-2024, NVIDIA CORPORATION & AFFILIATES.  # noqa: E501
```

Comment:
I don't really want to make linter exceptions for the copyrights "inline". 😕 What alternatives do we have? A line length of 88 would permit this.

Reply:
Almost everywhere else (which is why we don't need it), we have `NVIDIA CORPORATION` rather than `NVIDIA CORPORATION & AFFILIATES`.

Not saying which is right/wrong, just noting.

Reply:
For these files, it needs to stay as-is with the "& AFFILIATES".